v2/go.sum

```diff
@@ -28,6 +28,7 @@ github.com/zmap/zcrypto v0.0.0-20191112190257-7f2fe6faf8cf h1:Q9MiSA+G9DHe/TzG8p
 github.com/zmap/zcrypto v0.0.0-20191112190257-7f2fe6faf8cf/go.mod h1:w7kd3qXHh8FNaczNjslXqvFQiv5mMWRXlL9klTUAHc8=
 github.com/zmap/zcrypto v0.0.0-20200513165325-16679db567ff h1:0DDYlvtXPb8EMtQPZ2TJDcM+adqtzy77QOndkCW79JQ=
 github.com/zmap/zcrypto v0.0.0-20200513165325-16679db567ff/go.mod h1:TxpejqcVKQjQaVVmMGfzx5HnmFMdIU+vLtaCyPBfGI4=
+github.com/zmap/zlint v1.1.0 h1:Vyh2GmprXw5TLmKmkTa2BgFvvYAFBValBFesqkKsszM=
```

Comment:
I don't think this is right, is it?

Reply:
To be honest, this was done by the "go" toolchain automatically and I simply committed it, even if I don't really understand, what it does. If you think, we don't need it, I'll revert it.

Reply:
I think this part of the diff should be reverted. It doesn't make sense for the module to depend on itself (and at an older version!)

Reply:
reverted